docs/UserGuide.md

```diff
@@ -69,7 +107,7 @@ Alternatively, you may read our [Developer Guide](https://ay2223s1-cs2103t-w10-3
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Overview of TrackAScholar app
+## Overview of TrackAScholar app - This is how the application will look like when opened
```

Comment:
I think this line should be in a new line, so that the header is just "overview of TAS"

Reply:
Ok sure thank you.I have moved this below